spec/models/card_spec.rb

```diff
@@ -26,7 +26,7 @@
   describe '#check_answer' do
     let(:answer) { 'Привет' }
 
-    it 'user answer equal original text' do
+    it 'users answer equal original text' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.